spec/controllers/ems_container_controller_spec.rb

```diff
@@ -71,7 +71,7 @@
         EvmSpecHelper.create_guid_miq_server_zone
         # set kubeclient to return a mock route.
         allow(Kubeclient::Client).to receive(:new).and_return(mock_client)
-        expect(mock_client).to receive(:discover)
+        expect(mock_client).to receive(:discover).twice
```

Comment:
I'd rather go with:
```suggestion
        expect(mock_client).to receive(:discover).at_least(:once)
```
as this is a mock and the amount of `discover` might change in the future.

Reply:
Also if we're using `allow` above, it's pointless to have an `expect` here...

Reply:
Fixed.